applications/esm/Dockerfile.esm

```diff
@@ -0,0 +1,44 @@
+# Accept the base image name as an argument
+ARG BASE_IMAGE
+
+# Extend from the base image passed as an argument
+FROM ${BASE_IMAGE}
+
+USER $SERVICE_NAME
+# Create Conda Env for ESM
+WORKDIR /app/
+RUN ${HOME}/conda/bin/mamba env create -f env.yml
+
+WORKDIR /app/esm
+RUN bash -c "source ${HOME}/conda/etc/profile.d/conda.sh && \
+source ${HOME}/conda/etc/profile.d/mamba.sh && \
+mamba activate esm_py11 && \
+pip install . &&\
+pip install torch==2.4.0+cpu torchvision==0.19.0+cpu torchaudio==2.4.0+cpu --index-url https://download.pytorch.org/whl/cpu  && \
+pip install intel-extension-for-pytorch==2.4.0"
+
+RUN echo "#!/bin/bash" >> /app/init.sh && \
+echo "source ${HOME}/conda/etc/profile.d/conda.sh" >> /app/init.sh && \
+echo "source ${HOME}/conda/etc/profile.d/mamba.sh" >> /app/init.sh && \
+echo "mamba activate esm_py11" >> /app/init.sh && \
+chmod +x /app/init.sh && \
+echo "source /app/init.sh" >> ~/.bashrc
+
+WORKDIR /app/esm/examples/lm-design/
+RUN wget https://dl.fbaipublicfiles.com/fair-esm/examples/lm_design/linear_projection_model.pt
```

Comment:
Is this needed ? If we do not download, does it automatically download when we run the program that uses this model ?